src/core/sagas/data/DataSagas.js

```diff
@@ -183,11 +184,12 @@ function* deleteEntitiesWorker(action :SequenceAction) :Generator<*, *, *> {
 
       const deleteRequests = value.map((dataObject) => {
 
-        const { entitySetId, entityKeyIds } = dataObject;
+        const { block, entitySetId, entityKeyIds } = dataObject;
         return call(deleteEntityDataWorker, deleteEntityData({
           entityKeyIds,
           entitySetId,
           deleteType: DeleteTypes.SOFT,
+          block: block || false,
```

Comment:
Do you really need the condition? Is there any place in the codebase that sets `block` to true?

Reply:
you're right, i guess it's not necessary at the moment, but i added the condition in case there is a delete request in the future that needs `block` to be true... i can definitely take it out if you think that makes more sense.